0021-protocol-naming.adoc

```diff
@@ -0,0 +1,46 @@
+= Introducing a protocol naming scheme
+Thomas Eizinger <thomas@coblox.tech>; Daniel Karzel<daniel.karzel@coblox.tech>
+:toc:
+:revdate: 2020-01-20
+
+NOTE: Author: {authors} +
+Date: {revdate} +
+Tracking issue: https://github.com/comit-network/RFCs/issues/130[#130]
+
+== Proposal
+
+We propose the following protocol names for Alpha and Beta ledger:
+
+- `HAN` - HTCL for Asset on Native
+- `HErc20` - HTLC for ERC20
+- `HOmni` - HTLC for OMNI
+- `HALigth` - HTLC for Lightning
```

Comment:
Is this meant to be 'HTLC for Asset on Lighting'?  Perhaps 'HAL'?

Reply:
we had `HAL` in the beginning, but that is limiting if you assume that another special implementation for a ledger starting with `L` could exist. For specific implementation names we opted for longer names.

Reply:
Reasonable point.